hawc/apps/animal/models.py

```diff
@@ -1235,6 +1235,13 @@ def percentControl(data_type, egs):
                 low = eg["lower_ci"]
                 high = eg["upper_ci"]
 
+            elif data_type == constants.DataType.DICHOTOMOUS:
+                if i == 0:
+                    control_incidence = eg["incidence"]
+                    n_1 = eg["n"]
+                if control_incidence is not None and control_incidence > 0:
+                    incidence = eg["incidence"]
+                    mean = ((incidence - control_incidence) / n_1) * 100
```

Comment:
i think this need to be ((i1/n1)-(i0/n0))/(i0/n0), where i is incidence and n is num animals